src/test/java/com/apifest/doclet/tests/DocletTest.java

```diff
@@ -0,0 +1,141 @@
+package com.apifest.doclet.tests;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.apifest.doclet.Doclet;
+
+public class DocletTest {

```

Comment:
I'm happy you have added unit tests :+1:

IMO that is a nice practice (TDD is even better ;)